main_app/templates/items/detail.html

```diff
@@ -48,9 +49,6 @@ <h3>{{ item.name }}'s Categories</h3>
     <h5>No categories</h5>
     {% endif %}
   </div>
-  Available categories will come after this line
```

Comment:
This was the line that was causing the "available categories blah blah" to show on the right of the screen. Deleted it.